code/game/machinery/doors/airlock.dm

```diff
@@ -1165,7 +1182,7 @@
 	sleep(1)
 	set_opacity(0)
 	update_freelook_sight()
-	sleep(4)
+	sleep(2.5)
```

Comment:
You also need to change the sleep in `close()`. Might be a good idea to replace the values with a var on the airlock so people don't forget about this if we tweak it in the future.

Reply:
var for both sleep in open() and close() also implemented